manifests/base/params.env

```diff
@@ -16,3 +16,4 @@ odh-tensorflow-gpu-notebook-image-n-2=quay.io/opendatahub/notebooks@sha256:fc52e
 odh-trustyai-notebook-image-n=quay.io/opendatahub/workbench-images@sha256:636fdf26877be4013bc516c5ef4a818b5f7f29119ec3f601005fefc6d49433f0
 odh-trustyai-notebook-image-n-1=quay.io/opendatahub/workbench-images@sha256:9c7dfc1c58bcbafbb4ffae58641e1bc4c3b5850886960bf27c652b7d0635d779
 odh-habana-notebook-image-n=quay.io/opendatahub/workbench-images@sha256:b0821ae2abe45387a371108ac08e7474b64255e5c4519de5da594b4617fd79fe
+odh-habana-notebook-image-n=quay.io/opendatahub/workbench-images@sha256:b0821ae2abe45387a371108ac08e7474b64255e5c4519de5da594b4617fd79fe
```

Comment:
is this for` odh-habana-notebook-image-n` ?

Reply:
and i guess it is not part of the validation but to fix/update?

Reply:
Hi @zdtsw, it's neither the fix nor update - this change is here just to trigger the GitHub action since it is activated only in case that `manifests/base/params.env` file is updated by the PR. So this commit won't be part of the PR once I raise it properly against upstream repo :slightly_smiling_face:

Reply:
ok i see, you are trying to make a negative test case by adding the same name in the file.